content_sync/pipelines/definitions/concourse/site-pipeline.yml

```diff
@@ -72,55 +83,79 @@ jobs:
         trigger: false
         on_failure:
           try:
-            put: ocw-studio-webhook
-            timeout: 1m
-            params:
-                text: |
-                  {
-                    "version": "((pipeline_name))",
-                    "status": "errored"
-                  }
+            do:
+            - put: ocw-studio-webhook
+              timeout: 1m
+              params:
+                  text: |
+                    {
+                      "version": "((pipeline_name))",
+                      "status": "errored"
+                    }
+            - put: slack-webhook
+              timeout: 1m
+              params:
+                alert_type: failed
+                text: "Failed to get webpack-json"
```

Comment:
Good suggestion! It is actually already there. In this case, the name of the pipeline is "failing_pipeline_test" because I knew it would fail and generate a notification. Really this is just some random test course someone made in QA that I populated. I think in reality it would be something like `live:the-name-of-the-course/build-ocw-site`

<img width="705" alt="Screen Shot 2022-05-18 at 3 12 23 PM" src="https://user-images.githubusercontent.com/41908054/169138280-900c94ce-0027-4b3f-8cf7-95546650e849.png">